specification/cognitiveservices/data-plane/Face/stable/v1.0/examples/CreateNewFaceList.json

```diff
@@ -6,7 +6,8 @@
     "faceListId": "sample_face_list",
     "body": {
       "name": "sample_list",
-      "userData": "User-provided data attached to the face list"
+      "userData": "User-provided data attached to the face list",
```

Comment:
[Nit] Let's unify with a trailing period. Same for others.

Reply:
Done.